editor/import/audio_stream_import_settings.cpp

```diff
@@ -543,7 +543,6 @@ AudioStreamImportSettingsDialog::AudioStreamImportSettingsDialog() {
 	loop_hb->add_child(memnew(Label(TTR("Offset:"))));
 	loop_offset = memnew(SpinBox);
 	loop_offset->set_max(10000);
-	loop_offset->set_step(0.001);
```

Comment:
I don't have a compiler right now to check, but isn't the default step 1.0? If so, this would make it so that you can only loop on full seconds, wouldn't it?

Reply:
Indeed, https://docs.godotengine.org/en/stable/classes/class_range.html#class-range-property-step says default is `0.01`. Let me check if I can do anything about this…

Reply:
The docs may be inaccurate, because in the code it says `double step = 1.0;` :laughing:

Reply:
Oh, right! That's definitely a documentation bug then. Looks like 1.0 has been the actual default value for years.